packages/x-components/src/components/__tests__/base-grid.spec.ts

```diff
@@ -1,18 +1,19 @@
-import { Identifiable } from '@empathyco/x-types';
 import { mount, Wrapper, WrapperArray } from '@vue/test-utils';
```

Comment:
In the `selected-filters-list.vue` component, we are also using the `filter.facetId` as the slot name. For consistency's shake, we should change it. 

However, as this is a property that the developer users of the library can modify through the adapter, and we haven't yet set any standards about how a facet id should be for us, I'm not sure if we should change it before deciding this.

Reply:
Same happens in the `facets.vue` component

Reply:
I would take a decision in this PR, if not this is the classic change that we will never do. Either we force to write kebab case Id's for the facets, or we transform them just for these components.

Reply:
lets do it! kebabCase also with the `facetId`.